www/postfixadmin/Portfile

```diff
@@ -36,8 +35,41 @@ destroot {
     file copy {*}[glob ${workpath}/${distname}/*] ${docpath}/html
 }
 
+default_variants    +mysql
+
+variant mysql description "use mysql8 for the db backend" {
+    depends_run-append \
+                    port:mysql8 \
+                    port:php73-mysql
```

Comment:
Given the number of times you reference the php version, you might want to set a variable for `${php_version}` and substitute that wherever the text "php73" appears currently. That way the next time the php version changes, it is easy to implement.